sklearn/linear_model/logistic.py

```diff
@@ -711,10 +711,20 @@ def logistic_regression_path(X, y, pos_class=None, Cs=10, fit_intercept=True,
     n_iter = np.zeros(len(Cs), dtype=np.int32)
     for i, C in enumerate(Cs):
         if solver == 'lbfgs':
+            if  0 < verbose <= 1:
+                iprint = 50
+            elif 1 < verbose <= 2:
+                iprint = 1
```

Comment:
reading the doc https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.fmin_l_bfgs_b.html it seems iprint=99 is also a specific option.

Reply:
Right. Every value between 0 and 101 is a valid option with difference output. I think the question is, given how verbose should match the higher to more output, how best to map verbose to iprint. The current behavior only allows two options (iprint=-1 and iprint=0). The proposed change is to allow some more verbosity but not as much granularity.

Do you think it is preferred to provide the full granularity of iprint? If so, could map
```
if verbose <= 0:
    iprint = -1
elif 0 < verbose < 100:
    iprint = (100 - verbose)
else:
  iprint = verbose
```

This would provide full access to the range of output from iprint, but much the higher verbose = more output

Reply:
I like option you proposed here so far so verbose=0,1,2,3 etc like we do everywhere.